src/main/java/org/cactoos/list/LengthOf.java

```diff
@@ -44,9 +45,17 @@
 
     /**
      * Ctor.
+     * @param items The array
+     */
+    public LengthOf(final Iterable<?> items) {
+        this(items.iterator());
```

Comment:
@englishman this is against one of our fundamental principles: no code in constructors